oci/oathkeeper/image.yaml

```diff
@@ -0,0 +1,16 @@
+version: 1
+
+release:
+  latest:
+    end-of-life: "2023-06-22T00:00:00Z"
+    edge: 0.40.3
+
+upload:
+  - source: "canonical/oathkeeper-rock"
+    commit: 7c09e02d6a23c9b671f557adb2023fed06be7c86
+    directory: .
+    release:
```

Comment:
you're missing the canonical track here - `<version>-base`. This is mandatory as all images must have at least that track, for consistency. See https://github.com/canonical/oci-factory#image-trigger-file (`upload[*].release.<track>`)